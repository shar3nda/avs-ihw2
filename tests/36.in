k	X@#=qDKp^h !a9P;
=gTUx~e"xI>k6fDZ12_k4ew,-xnP*1xRaE[]__#?W*3X%1Z^'vI:<WXwe)?