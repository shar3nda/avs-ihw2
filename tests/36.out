Lowercase: 295, uppercase: 310